syft/workers/websocket_client.py

```diff
@@ -8,6 +8,7 @@
 import logging
 import ssl
 import time
+import pyshark
```

Comment:
This is a great work. It looks very good and it will be very helpful.
My only concern is whether this utility could be somewhere else and not within this class.

Reply:
Yeah, I could move them to somewhere more appropriate... Any suggestion(s)?

Reply:
I could move the `pyshark` utility elsewhere, say further up like at [syft/workers/base.py](https://github.com/OpenMined/PySyft/blob/1d216ae362bc35a2238cbaf12a9c02f047debefd/syft/workers/base.py)? Would that be more appropriate? I checked `torch` is imported there hence my suggestion.

Reply:
I would move it to its own file/module and let the users that want to have those metrics import it in their projects.
What about `metrics.py`?
Second opinions are welcomed!

Reply:
It depends a bit of how it is used I would say, if you have an example to share it would be super cool!
I would say, either this is very related to workers is that case we can have it as part of the abstract.py worker file, or if it is more agnostic of the concept of worker, then a `metrics.py` file maybe in syft/generic or syft/federated would be more appropriate!

Reply:
Thanks for the suggestions! I have moved the utility to a spot called `metrics.py` in the `PySyft/syft/generic` directory. I think this would be a lot more fitting than its former location. Also, I will add examples and explanations for the newly added methods soon, say within the next few days.